src/showcase/stories/angular/sidebar/sidebar-title/sidebar-title.stories.ts

```diff
@@ -1,9 +1,30 @@
 import { SbbSidebarTitle } from '@sbb-esta/lyne-angular/sidebar/sidebar-title';
 import { Args, argsToTemplate, Meta } from '@storybook/angular';
+import { ArgTypes, InputType } from '@storybook/types';
+
+const level: InputType = {
```

Comment:
level and visualLevel are internal property, so I'd prefer to not have them documented

Reply:
same as before, controls are automatically generated; should i remove them all?